test/integration/api.bats

```diff
@@ -172,9 +198,24 @@ function teardown() {
 	skip
 }
 
-# FIXME
 @test "docker rename" {
-	skip
+	start_docker 3
+	swarm_manage
+
+	run docker_swarm run -d --name test_container busybox sleep 500
+	[ "$status" -eq 0 ]
+
+	# rename container
+	run docker_swarm rename test_container rename_container
+	[ "$status" -eq 0 ]
+
+	# cluster refresh state of container during 30 seconds	
+	sleep 35
```

Comment:
Actually, I think we should also force refresh container when rename, what do you think about this?